cmd/go-getter/main.go

```diff
@@ -10,6 +10,7 @@ import (
 
 func main() {
 	modeRaw := flag.String("mode", "any", "get mode (any, file, dir)")
+	progress := flag.Bool("progress", false, "display terminal progress")
```

Comment:
I see why `gopkg.in/cheggaaa/pb.v1` is being included now.

Can we move this down to this package (or even a package below `cmd/go-getter`) so that it's only used when the CLI needs it?